blitzstudios/sleeper-mini-core
# Synchronizing package dependencies with main sleeper app.

### Description

This commit aligns all sleeper-mini-core package versions with our main sleeper app. This fixes a few things:
1) Fast image + interactable bugs
2) A missing react-refresh-webpack-plugin plugin (necessary for using the inspector tool in react native)

### How To Test

1. Launch the app.
2. Shake the device, and open the inspector tool.
3. Make sure no crash occurs.
